resources/scripts/ttx_diff.py

```diff
@@ -508,6 +510,15 @@ def remove_mark_and_kern_lookups(ttx):
             lookup.remove(child)
         if is_extension:
             lookup_type_el.attrib["value"] = str(lookup_type)
+        removed_indices.add(lookup_index)
+
+    # finally go and replace any of the removed indices where they are referenced
+    # in features; our lookup order won't always match fonttools for mark/kern
+    # because of the particulars of the feature writers, so we just ignore this
```

Comment:
This should address why this isn't hiding problems. Do we compare the normalized mark/kern data so this doesn't matter? - then this should explicitly say so

Reply:
yes, these lookups are all normalized and normalization accounts for lookup order. Will clarify the comment.